packages/manager/src/features/CloudPulse/Utils/FilterConfig.ts

```diff
@@ -3,11 +3,14 @@ import { CloudPulseSelectTypes } from './models';
 import type { CloudPulseServiceTypeFilterMap } from './models';
 
 const TIME_DURATION = 'Time Range';
+const DBAAS_CAPABILITY = 'Managed Databases';
+const LINODE_CAPABILITY = 'Linodes';
 
 export const LINODE_CONFIG: Readonly<CloudPulseServiceTypeFilterMap> = {
   filters: [
     {
       configuration: {
+        capability: LINODE_CAPABILITY,
```

Comment:
Do we need to define this capability at every filter for a service?  Can we define once at service level?  @venkymano-akamai   any thoughts?

Reply:
@ankita-akamai push the capability key outside the configuration at the service level, because it is more related to service type & has nothing to do with individual filters